modules/modtcl/modtcl.tcl

```diff
@@ -34,6 +34,7 @@ set ::botnet-nick ZNC_[GetUsername]
 set ::botnick [GetCurNick]
 set ::server [GetServer]
 set ::server-online [expr [GetServerOnline] / 1000]
+set ::network	[GetServerName]
```

Comment:
What exactly did you change compared to the previous PR? Looks exactly the same for me: GetServerName vs GetNetworkName.

I'd prefer GetNetworkName, because server name is probably a domain name, or IP address.